RHEL6_7/system/initscripts/ifcfg/preupg_network.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+orig_name_base="eth"
+new_name_base="net"
+conf_dir='/etc/sysconfig/network-scripts'
+temp_conf_dir='/root/preupgrade/dirtyconf/etc/sysconfig/network-configuration_fixed/'
+preupg_script='/root/preupgrade/preupgrade-scripts/rename_network.sh'
```

Comment:
@jmazanek: you would be able to use here probably API variables. it would be better use them when it works. Still you can export them if these are not set.